config/nimbus.yaml

```diff
@@ -47,17 +47,3 @@ features:
         value: { "enabled": true }
       - channel: production
         value: { "enabled": false }
-  last-scan-date-csat-survey:
```

Comment:
Removed in favor of the feature flag `LatestScanDateCsatSurvey` so we can enable/disable the CSAT survey easily for users who are/aren’t part of the `last-scan-date` experiment.